tests/rogo_test/parse_question_test.py

```
@@ -124,7 +137,7 @@ def test_parse_question():
         assert type(output) is ParseWordLatToCompQuestion
 
         assert output.check(output.main_answer)
-        ic(output)  # type: ignore[name-defined] # noqa: F821
+        ic(output)  # noqa: F821
```

Pylint (3.3.7)

[error] Undefined variable 'ic'

(E0602)